### PR TITLE
ref(vue): Add vue-router version support information

### PR DESCRIPTION
**Pull request by Lms24**
This adds a note on which `vue-router` versions are supported by our vue routing instrumentation. Also did some minor cleanup of the code example as it contained some unnecessary lines. 

ref: https://github.com/getsentry/sentry-javascript/issues/5910

### CODE REVIEW

**Review by AbhiPrasad — APPROVED** [review]

File: src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx [hunk 1]
```diff
@@ -5,21 +5,24 @@ redirect_from:
   - /sdks/react/integrations/vue-router/
 ---
 
-If you are using Vue Router, you should use our provided integration to get better, parameterized transaction names. Here is a full example of how to use it:
+If you are using Vue Router, you should use our provided routing instrumentation to get better, parameterized transaction names.
```
Comment (imatwawana) [inline src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx:8]:
```suggestion
If you're using Vue Router, you should use our provided routing instrumentation to get better, parameterized transaction names.
```

File: src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx [hunk 2]
```diff
@@ -5,21 +5,24 @@ redirect_from:
...
 
-```javascript
+Our routing instrumentation supports `vue-router` v2, v3 and v4.
```
Comment (imatwawana) [inline src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx:10]:
```suggestion
Our routing instrumentation supports `vue-router` v2, v3, and v4.
```

**Review by imatwawana — APPROVED** [review]
A couple style edits

File: src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx (hunk 1 above)
Comment (Lms24) [reply src/platforms/javascript/guides/vue/configuration/integrations/vue-router.mdx:8]: There'll come a day when I remember this but the last 12 years of uni/school telling me not to use these abbreviations still have their influence on me. So thanks for always correcting these ;)

**Pull request merged**